workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
now I recall why I did that..
if you just use the ${i} it will run 4 times with the number 4.
groovy is mutating the value
and by the time the closure runs it will execute 5 times with the same parameter - the number 4
you have to assign it outside the closure

see the following explanation:
http://stackoverflow.com/questions/32228590/start-a-job-multiple-times-concurrently-with-workflow-plugin